rdl/rdl-gen-athenz-go-client/main.go

```diff
@@ -116,21 +116,19 @@ var _ = ioutil.NopCloser
 type {{client}} struct {
 	URL             string
 	Transport       http.RoundTripper
-	CredsHeader     *string
-	CredsToken      *string
+	credsHeaders    map[string]string
```

Comment:
since all other fields can be exported let's be consistent and name this CredsHeaders as well.